CONTRIBUTING.org

```diff
@@ -0,0 +1,186 @@
+the filename.
+
+Recipes should try to minimize the size of the resulting package by
+specifying only files relevant to the package.
+
+*** Test your recipe
+
+Please test that the package builds properly by following the steps
+below.
+
+Let ~<NAME>~ denote the filename of the new recipe. Build the recipe
+via ~make recipes/<name>~, or through pressing ~C-c C-c~ in the recipe
+file buffer. Be sure that the ~emacs~ binary on your ~PATH~ is at
+least version 23, or set ~$EMACS_COMMAND~ to the location of a
+suitable binary.
+
+If the repository contains tags for releases, confirm that the correct
+version is detected by running ~STABLE=t make recipes/<NAME>~.  The
+version detection can be adjusted by specifying ~:version-regexp~ in
+the recipe (see [[file:README.md#recipe-format][recipe format]] in the README).
+
+Test that the package installs properly by running
+~package-install-file~ from within Emacs and specifying the newly
+built package in the directory specified by
+~package-build-archive-dir~ (default: ~packages/~ directory where
+~package-build~ was loaded). Entering "yes" when prompted after
+pressing ~C-c C-c~ in the recipe buffer also works.
+
+You can optionally run a sandboxed Emacs in which locally-built
+packages will be available for installation along with those already
+in MELPA:
+
+#+BEGIN_SRC shell
+EMACS_COMMAND=/path/to/emacs make sandbox INSTALL=<name>
+#+END_SRC
+
+From within Emacs, install and test your package as appropriate. This
+is a useful way to discover missing dependencies.
+
+** Opening a pull request
+
+Create a [[https://github.com/magit/magit/wiki/Dedicated-pull-request-branches][dedicated pull request branch]] in your clone of the [[https://github.com/melpa/melpa][MELPA
+repository]] and push this branch to your fork. Finally, go to the MELPA
+repository and open the pull request.
+
+Include the following information in the pull request description:
+
+- a brief summary of what the package does;
+- a direct link to the package repository;
+- your association with the package (e.g., are you the maintainer?
+  have you contributed? do you just like the package a lot?);
+- relevant communications with the upstream package maintainer (e.g.,
+  ~package.el~ compatibility changes that you have submitted).
+
+Consider the [[https://github.com/github/hub][hub]] command-line utility by [[http://chriswanstrath.com/][defunkt]] which helps simplify
+this process.
+** Waiting for reviews and taking feedback into account
+
+MELPA maintainers spend a lot of time reviewing proposed packages and
+also have quite a lot of other non-MELPA-related activities. Please be
+patient as it might take a week (sometimes several) before one starts
+having a look at your pull request. 
+
+You can help MELPA maintainers take care of pull requests much faster
+by paying real attention to the quality of your package (see above for
+some quality checks and links). If you feel for it, you can also take
```

Comment:
"If you were asked to make several changes, then you should explicitly mention everything that you have fixed, and possibly even link to the relevant commits. Once way of doing that is to mention the melpa pull-request in every commit that addresses one of the raised points, using the format `melpa/melpa#N`."

(Double check that `melpa/melpa#N` works.)

Reply:
Thanks for the paragraph. I introduced it.

> Double check that melpa/melpa#N works

done. It does